docker-compose.yml

```diff
@@ -28,6 +31,62 @@ services:
       BIGCHAINDB_DATABASE_PORT: 27017
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
       BIGCHAINDB_WSSERVER_HOST: 0.0.0.0
+      BIGCHAINDB_START_TENDERMINT: 0
+      BIGCHAINDB_TENDERMINT_HOST: tendermint
+      BIGCHAINDB_TENDERMINT_PORT: 46657
     ports:
       - "9984"
-    command: bigchaindb start
+      - "46658"
+    healthcheck:
+      test: ["CMD", "curl", "http://bigchaindb:9984"]
+      interval: 3s
+      timeout: 5s
+      retries: 3
+    command: bigchaindb -l DEBUG start
+  tendermint:
+    image: tendermint/tendermint:0.13
+    volumes:
+      - ./tmdata/config.toml:/tendermint/config.toml
+    entrypoint: ''
```

Comment:
is this needed to override the default entry point in tendermint image?

Reply:
yes.